core/federated/RTI/rti_common.c

```diff
@@ -10,6 +10,8 @@
 #if defined STANDALONE_RTI || defined LF_ENCLAVES
 #include "rti_common.h"
 
+extern lf_mutex_t rti_mutex;
```

Comment:
I dont think this is needed. 

In `initailize_rti`, both in `rti_remote.c` and `rti_local.c` the `mutex` field of the `rti_common` is set. So we should be able to just use this field?